### ISSUE TITLE
Pull price from multiple locations and average per day/week

### ISSUE THREAD

**Issue by tjayrush** [Great-Hill-Corporation/quickBlocks#147]
Accountants need price per block. At 14 second resolution, a transaction may be more advantagiously priced for a token purchase, or perhaps less advantagiously priced. The idea is to price the transaction at the average of 5 min, 10 min, 1 hour, per day that best advantages the client. One could do this per block, per minute, per hour, per half-day, per day whatever, but one would have to do this for all transactions in the reporting period to avoid 'price shopping.' So here's the algo: Price a list of transactions per block (14 seconds), then do exact same pricing per minute, then per 15 minutes, then per hour, then per half-day, then per day. Do this across all days. Whichever is the most advantagious pricing mode, use that for the whole period (month, year, etc.)

**Comment by tjayrush** [comment]
CoinCompare seems to be an amazing source of data

**Comment by tjayrush** [comment]
Closing for now. Will reopen once 0.2.0 is finished. Referenced by #293.

**Comment by tjayrush** [comment]
One day, just not now.
